package.json

```diff
@@ -52,6 +52,7 @@
     "@typescript-eslint/parser": "^5.51.0",
     "@vitejs/plugin-react": "^3.1.0",
     "@yarnpkg/sdks": "^3.0.0-rc.38",
+    "chokidar": "^3.5.3",
```

Comment:
Was this an optional dependency of some tool's watch mode or something? I don't see it called explicitly.

Reply:
Yeah, `@swc/cli`. I don't actually know why I added this, though, because we don't have any scripts that use swc's watch mode; everything seems fine if I remove it.